src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_account_commands.py

```diff
@@ -64,7 +64,7 @@ def test_list_accounts(self):
     @ResourceGroupPreparer(name_prefix='cli_netappfiles_test_account_')
     def test_get_account_by_name(self):
         account_name = self.create_random_name(prefix='cli', length=24)
-        account = self.cmd("az netappfiles account create -g {rg} -a %s -l %s" % (account_name, LOCATION)).get_output_in_json()
+        self.cmd("az netappfiles account create -g {rg} -a %s -l %s" % (account_name, LOCATION)).get_output_in_json()
```

Comment:
If you don't assign the result to any variable, then get_output_in_json can be removed.